README.md

```diff
@@ -22,6 +22,7 @@ The packages in this library implement the various components of the CONIKS syst
 - ``merkletree``: Merkle prefix tree and related data structures
 - ``keyserver``: Server-side protocol implementation
 - ``utils``: Utility functions 
+- ``protocol``: Reusable CONIKS server library and the server executable implementation
```

Comment:
``protocol`` also contains some client-side protocol code, no? And the server executable is in ``keyserver``? I would generalize the description here to something like "CONIKS protocols implementation/library"